funcx_endpoint/setup.py

```diff
@@ -40,6 +40,7 @@
     # of different features and functions
     # pin exact versions because it does not use semver
     "parsl==1.1.0",
+    "pika>=1.2.0",
```

Comment:
I've put in [a bug report that the policy in `pika` is undocumented](https://github.com/pika/pika/issues/1343), but it does appear to be calver. It may be prudent to write `<2`, or it might not. No change is strictly necessary, but I'll keep an eye on this.

Since `funcx-endpoint` is an application, not a library, I would tend to set some kind of upper bound for its dependencies.